htdocs/fourn/commande/list.php

```diff
@@ -291,7 +291,7 @@
 				$objecttmp->fk_project = $cmd->fk_project;
 				$objecttmp->multicurrency_code = $cmd->multicurrency_code;
 				if (empty($createbills_onebythird)) {
-					$objecttmp->ref_client = $cmd->ref_client;
+					$objecttmp->ref_supplier = !empty($cmd->ref_supplier) ? $cmd->ref_supplier : dol_print_date(dol_now(), '%Y%m%d%H%M%S');
```

Comment:
If we set a generated value as default value, it would be better if this value is the same for all the transaction.
So can you create the default ref outside of the loop ?

Reply:
@eldy in fact i was wrong, we do need to have a different default ref supplier because this field needs to be unique in supplier invoices.
I just did a new commit which seems better.